_pages/welcome-to-TTS/getting-started.md

```diff
@@ -0,0 +1,248 @@
+
+### Cloud.gov
+
+Just as Federalist makes creating .gov sites as easy as possible, [Cloud.gov ](https://cloud.gov/)makes hosting them as easy as possible, too, by dealing with a bunch of security and compliance sticking points so you don't have to. That convenience and trusted status means almost everything we build is hosted on cloud.gov, including Federalist, [cloud.gov is extensively documented](https://cloud.gov/docs/) and heavily supported in [#cg-support](https://gsa-tts.slack.com/messages/C09CR1Q9Z). Still not clear? [They have a video, too](https://www.youtube.com/watch?v=nCycNkxs-rE).
+
+### USWDS
+
+USWDS --- the [United States Web Design System](https://designsystem.digital.gov/) --- is a frontend framework designed to make it easy to build accessible, mobile-friendly government websites. They've worked through hundreds of issues related to accessibility, usability, and the overall user experience. Your project can leverage that effort nearly instantly. To try out the USWDS on a test site, you can use the [templates provided by Federalist](https://federalist.18f.gov/documentation/templates/).
+
+Those are just three of the products TTS is working on, but [there are many more](https://docs.google.com/document/d/1vfBiQsprY1m8dG4tS_GeXkhG6ThF5ZqW9JKlDlvbWis/edit).
+
+
+## Acquisitions and Procurement
+
+18F alumni Sasha Magee memorably summed up the progression through government digital service [like this](https://twitter.com/sashax/status/941036458307076097):  
+   Year 1: "Why y'all talk about procurement so much? How boring."  
+   Year 2: "Man, procurement's a pain."  
+   Year 3: "We should really consider doing something about procurement"  
+   Year 4: "Holy crap there's nothing more important than solving procurement!"
+
+You're going to hear a lot about acquisitions and procurement. For a quick dive, [start with this simple introductory presentation](https://docs.google.com/presentation/d/1FjJWwXrTbXexdTGeZKP6OjQ8i5B3GmOAj5MQTF_arLg/edit?disco=AAAADFGLhBo&ts=5cf90ee6&usp_dm=false#slide=id.g4eb21fd7af_0_178).You may also want to view these relevant decks presented during past 18F Team Coffees:
+
+-   [Pre-award management](https://docs.google.com/presentation/d/1uOuzFibFCvgW3I0XL54w2sKl7ObSBxk9cUlyTchJ8Gk/edit#slide=id.g57d5603040_0_23)
+
+-   [Post-award management](https://docs.google.com/presentation/d/1pBm3jTlkvPJyOX3KtbjZVcFkKrEwCyqJ9jdOXh0PPuA/edit#slide=id.g57d5603040_0_23)
+
+After that, dive into the relevant handbook pages or ask any questions you may have in [#acquisition](https://gsa-tts.slack.com/messages/C054NH1FW). If you want to go really deep there are [free courses available](https://www.fai.gov/certification/contracting-fac-c) from the Federal Acquisition Institute.
```

Comment:
Could we add a link to the handbook pages on procurement? This one is probably the best starting point: https://handbook.18f.gov/working-on-an-acquisition-engagement/

Reply:
Done!